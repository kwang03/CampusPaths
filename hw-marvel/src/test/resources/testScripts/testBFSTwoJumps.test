#Tests BFS where shortest path is two jumps away

LoadGraph myGraph twoJumps.csv

FindPath myGraph Char1 Char3