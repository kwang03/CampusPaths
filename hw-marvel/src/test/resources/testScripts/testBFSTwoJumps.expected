#Tests BFS where shortest path is two jumps away

loaded graph myGraph

path from Char1 to Char3:
Char1 to Char2 via Book1
Char2 to Char3 via Book2